src/fprime_gds/common/encoders/file_encoder.py

```diff
@@ -43,11 +43,11 @@
 
 import struct
 
-from fprime.common.models.serialize.numerical_types import U32Type
+from fprime_gds.common.models.serialize.numerical_types import U32Type
 from fprime.constants import DATA_ENCODING
```

Comment:
Ah! This is using the old `DATA_ENCODING` definition. Need to move to the new definition in fprime-gds